Gemfile

```diff
@@ -6,8 +6,7 @@ group :development do
   gem 'pry'
   gem 'pry-byebug'
   gem 'rubocop'
-  gem 'rack', '~> 1.6.8' if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.2')
-  gem 'smart_proxy', :github => "theforeman/smart-proxy", :branch => 'develop'
+  gem 'smart_proxy', :github => "theforeman/smart-proxy", :branch => ENV['SMART_PROXY_BRANCH']
```

Comment:
This was discussed this in another PR: it relies on a bundler 2.3 feature (shipped in Ruby 3.1+) to support default branches not named master. `ENV.fetch('SMART_PROXY_BRANCH', 'develop')` is needed if you want to support older versions. Shall I submit a PR to fix it up?

Reply:
https://github.com/theforeman/smart_proxy_openscap/pull/98

Reply:
https://github.com/theforeman/actions/pull/62#discussion_r1837652960 for more context.